lambda-notes/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
```

Comment:
Hi Xang! This is giving me an error when I try to load it, because it's not wrapped in a `<Provider>` tag. I would recommend deleting the test file. I'm not aware of any reason you'll need it... and if you do find you'll need it again, it will be easy enough to replace.